src/test/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheckTest.java

```diff
@@ -378,6 +381,58 @@ public void testCountMatches() throws Exception {
         assertEquals("Unexpected matches count", 1, actual);
     }
 
+    /**
+     * Testing, that all elements in the constant NON_PRINTABLE_CHARS are sorted.
+     * This is very convenient for the sake of maintainability.
+     *
+     * @throws Exception when code tested throws some exception
+     */
+    @Test
+    public void testNonPrintableCharsAreSorted() throws Exception {
+        // Getting Field Value via Reflection, becaues the field is private
+        final String fieldName = "NON_PRINTABLE_CHARS";
+        final Field field = AvoidEscapedUnicodeCharactersCheck.class.getDeclaredField(fieldName);
+        field.setAccessible(true);
+        String expression = ((Pattern) field.get(null)).pattern();
+
+        // Replacing expressions like "\\u000[bB]" with "\\u000B"
+        final String charExpressions = "Aa,Bb,Cc,Dd,Ee,Ff";
+        for (String charExpression : charExpressions.split(",")) {
```

Comment:
why can't we do `new String[]{"Aa", "Bb"` etc... instead of creating the string and then splitting it?